Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:bionic
+
+WORKDIR /srv/cartography
+
+ENV PATH=/venv/bin:$PATH
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . /srv/cartography
+
```

Comment:
Not sure if there is an appetite to harden this, but does it need to be run as `root`? 
Can we drop to an un-privileged user?